### DISCUSSION TITLE
Overlay Dissapears After Turning Off GoPro

### DISCUSSION THREAD

**Discussion by cherlishinsubordinate** [gopro/labs#814] [General]
I am trying to add time and GPS data overlay to video from a Hero10 Black. This is the QR code I used: g1!MLFIN=0!MBRNO=10!MBURN="(0,60)[TCHH:MM:SSaa mm-dd-yyyy 'GPS50%1.6f', 'GPS51%1.6f']". It works right after I scan the code, but if I turn the GoPro off, I have to rescan the code in order for the overlay to work. Is there a way to have the overlay permanently appear on video?

**Comment by dnewman-gpsw** [comment]
That is the permanent command.  It should report the overlay is active each time you boot. If it is not, start with a scan of !RESET (approve the reset) and repeat that command once again.

**Reply by cherlishinsubordinate** [reply]
I scanned !RESET and repeated the command but still no luck. It doesn't report overlay active.

**Reply by dnewman-gpsw** [reply]
Share your MISC/qrlog.txt file.  Also what does your camera display on boot?

**Reply by cherlishinsubordinate** [reply]
Attached is the file. On boot, it sometimes displays "GoPro labs beta firmware. Participation rules apply."
[qrlog.txt](https://github.com/gopro/labs/files/14619252/qrlog.txt)

**Reply by dnewman-gpsw** [reply]
Okay, HERO10 doesn't tell so much in default.  Scan !MDBGL=1 for more debug and report qrlog.txt again on the next boot.  Also it should say more than the beta message, it should report what is active.  With recent video capture, please report your metadata from https://gopro.github.io/labs/control/metadata/  That should show what Labs features are enabled.

**Reply by cherlishinsubordinate** [reply]
[qrlog.txt](https://github.com/gopro/labs/files/14619375/qrlog.txt)

Here is the metadata:

DEVC 
  DVID 1
  DVNM Global Settings
  VERS 8, 2, 2
  FMWR H21.01.01.50.70
  LINF LKO3011601400201
  CINF 69, 27, 9, -5, 42, 41, -29, 43, 73, -31, -112, 55, -4, -26, -76, 22
  CASN C3464280017755
  MINF HERO10 Black
  MUID -83289275, 736307498, 932241737, 380954364, 0, 0, 0, 0
  CMOD 12
  MTYP 0
  OREN U
  DZOM Y
  DZST 0
  SMTR N
  PRTN Y
  PTWB AUTO
  PTSH MED
  PTCL NATURAL
  EXPT AUTO
  PIMX 1600
  PIMN 100
  PTEV 0.0
  RATE .empty.
  VFOV L
  SROT 6.0660
  ZFOV 95.4312
  EISE Y
  EISA HS High
  HLVL N
  AUPT N
  AUDO AUTO
  BROD .empty.
  BRID 0
  PVUL F
  PRJT GPRO
  SOFF 0
  CLKS 1
  CDAT 0x0000000065F45963
  SCTM 0
  PRNA 58
  PRNU 0
  SCAP N
  CDTM 0
  DUST NO_LIMIT
  VRES 1920, 1080
  VFPS 30000, 1001
  HSGT OFF
  BITR STANDARD
  MMOD STEREO
  RAMP .empty.
  TZON 32767
  CLKC 0
  DZMX 2.0000
  PWPR PERFORMANCE
  ORDP Y
  CLDP Y
DEVC 
  DVID USRM
  DVNM User Metadata
  STRM 
    DBGL 1
    USRD 1
DEVC 
  DVID HLMT
  DVNM Highlights

**Reply by dnewman-gpsw** [reply]
The overlay metadata is missing. Only thing stored is DBGL. Try scan just

![image](https://github.com/gopro/labs/assets/10856084/32db3b4d-35db-4fac-8ddf-f6500a890b37)

**Reply by cherlishinsubordinate** [reply]
That worked, thanks! What's different about this QR?

**Reply by dnewman-gpsw** [reply]
Best guess, your browser is adding a character that is not savable.  ```' and "```  characters have the standard ASCII 7-bit versions, some keyboards, browser, or languages can inject unicode characters.

**Reply by cherlishinsubordinate** [reply]
Interesting. Thanks for the help.
